pii-manager/src/pii_manager/piientity.py

```diff
@@ -0,0 +1,20 @@
+from .piienum import PiiEnum
+
+
+class PiiEntity:
+    """
+    A detected PII entity
+    """
+
+    __slots__ = "pos", "elem", "value"
+
+    def __init__(self, elem: PiiEnum, pos: int, value: str):
+        self.elem = elem
+        self.pos = pos
```

Comment:
The position only defines the start index in the string. Shouldn't we add the stop index?

Reply:
well, it's either start+end or start+length. In this case I opted for the second, by adding a `__len__` method to the class, so that one can simply say e.g.  `obj.pos +  len(obj)`. But we can also add an endpointm if you feel it's needed